wire-ios-data-model/Source/Model/UserClient/UserClient.swift

```diff
@@ -416,6 +416,7 @@ public extension UserClient {
         client.model = model
         client.deviceClass = deviceClass.map { DeviceClass(rawValue: $0) }
         client.activationDate = activationDate
+        client.lastActiveDate = lastActiveDate
```

Comment:
Please check this change. There was a warning before, because the parsed date wasn't used.

Reply:
Seems like mistake. I suppose it was always nil until now. 

This is used in the  `SupportedProtocolsService` to calculate the self supported protocols and it being null means that old abandoned clients were preventing MLS from becoming a supported protocol.